ui/app/components/clients/page/sync.hbs

```diff
@@ -2,58 +2,68 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 ~}}
+{{#if @secretsSyncActivated}}
+  {{#if this.isDateRange}}
+    <Clients::ChartContainer
+      @title={{this.title}}
+      @description={{this.description}}
+      @timestamp={{@activity.responseTimestamp}}
+      @hasChartData={{this.byMonthActivityData}}
+      class="no-legend"
+    >
+      <:subTitle>
+        <StatText
+          @label="Total sync clients"
```

Comment:
@hellobontempo maybe a question for Yankun, but I feel like the language here is confusing. Should it instead be `Total Secret Sync clients`? Not something we need to address for GA.

Reply:
This was intentional because we aren't referring to the feature here, but the sync clients as a noun and the casing matches the other `StatText` items. I believe `secrets` was intentionally removed here for brevity because we're on the sync page.